Formula/goreleaser.rb

```diff
@@ -14,10 +14,16 @@ class Goreleaser < Formula
   depends_on "go" => :build
 
   def install
-    ENV["GOPATH"] = HOMEBREW_CACHE/"go_cache"
-    (buildpath/"src/github.com/goreleaser/goreleaser").install buildpath.children
-    cd "src/github.com/goreleaser/goreleaser" do
-      system "go", "build", "-ldflags", "-X main.version=#{version}",
+    ENV["GOPATH"] = buildpath
+    ENV["GO111MODULE"] = "on"
```

Comment:
Can you please describe what this env var does?

Reply:
go modules are new on go 1.11, and are behind this flag, which by default is `auto`.

on `auto`, when the project is inside GOPATH, it will disable go modules, and will always be `on` otherwise.

setting this to on is only to make sure that it will always be enabled.

Reply:
For homebrew, I think it defaults to use `go 1.12`, which defaults to use `go modules`, I don't see why we need this option?